astrolabe/cli.py

```diff
@@ -506,5 +536,11 @@ def validate_workload_executor(workload_executor, startup_time,
         exit(1)
 
 
+@spec_tests.command('stats')
+@click.pass_context
+def stats(ctx):
```

Comment:
Do we call this anywhere or is it a utility for developing/debugging? If it is only a utility, shouldn't we also print out these stats instead of just writing them to a file? (assuming that developing/debugging workflows would usually involve interacting with Astrolabe in a shell)

Reply:
Astrolabe aggregates the statistics so that workload executors are not obligated to do so. The statistics are uploaded as an artifact for each  test run. I personally haven't needed to see these in the terminal as part of log output which is what I think you are suggesting.